plugin/docker.go

```diff
@@ -84,7 +84,7 @@ func commandBuild(build Build, dryrun bool) *exec.Cmd {
 	}
 
 	args = append(args, build.Context)
-	if !dryrun && len(build.Tags.Value()) > 0 {
+	if !dryrun && build.Output == "" && len(build.Tags.Value()) > 0 {
```

Comment:
I think this should be `if (!dryrun && len(build.Tags.Value()) > 0) || build.Output == ""`

Reply:
I had a second look and I think your current condition is correct sorry.